prime-router/src/main/resources/db/migration/V11__add_test_data_table.sql

```diff
@@ -0,0 +1,61 @@
+/*
+This SQL creates the tables of the DB. The Flyway tool applies this migration to create the database
```

Comment:
if this file is hand-authored, then _consider_ adding a `check` constraint to `varchar` fields that are nullable along the lines of
`fieldname varchar(42) null check (fieldname is null or lenght(fieldname) > 0)`
This will prevent empties from making it into the database (and yes, requiring you to substitute empties in the source for `null`s)

This suggestion **only** applies if there is no semantic difference between "not present" (i.e. `null`) and "empty" (i.e. `''`).